### PR TITLE
[MOD-2610] fix(example): merge example of ios and android into root directory

### PR DESCRIPTION
**Pull request by akansha-maheshwari-axway**
https://jira.appcelerator.org/browse/MOD-2610

### CODE REVIEW

**Comment by akansha-maheshwari-axway** [comment]
@fasihul-axway @aakash-jain-axway @vikas-goyal-axway @anil-shukla-axway @garymathews 
Can you please review this PR?

File: ios/titanium.xcconfig [hunk 1]
```diff
@@ -4,17 +4,13 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
-TITANIUM_SDK_VERSION = 9.0.0.v20200205142057
+TITANIUM_SDK_VERSION = 9.0.0.GA
```
Comment (fasihul-axway) [inline ios/titanium.xcconfig:7]: Create  another separate PR for configure file changes.

File: Example/app.js [hunk 2]
```diff
@@ -6,77 +6,97 @@
  *
  */
 var TiIdentity = require('ti.identity');
-
+var isandroid = Ti.Platform.name === 'android';
```
Comment (aakash-jain-axway) [inline Example/app.js:9]: typo. Name should be isAndroid

File: Example/app.js [hunk 3]
```diff
@@ -6,77 +6,97 @@
...
 
-	if (!supported) {
-		alert('Authentication is not supported. Available biometrics: ' + authPhrase);
+if (isiOS) {
+	if (TiIdentity.biometryType === TiIdentity.BIOMETRY_TYPE_FACE_ID) {
+		authPhrase = 'Face ID';
+	} else if (TiIdentity.biometryType === TiIdentity.BIOMETRY_TYPE_TOUCH_ID) {
+		authPhrase = 'Touch ID';
+	} else {
+		authPhrase = '(None available)';
```
Comment (aakash-jain-axway) [inline Example/app.js:38]: the none available constant can be created into a string constant variable and used for both android & ios check.

Comment (akansha-maheshwari-axway) [reply Example/app.js]: Done

File: Example/app.js (hunk 2 above)
Comment (akansha-maheshwari-axway) [reply Example/app.js]: Done.

**Comment by vikas-goyal-axway** [comment]
Looks Good 👍

**Pull request merged**